sysutil/filesystem.go

```diff
@@ -29,7 +30,7 @@ func (fs *FileSystem) ReadLines(filePath string) ([]string, error) {
 	var result []string
 	s := bufio.NewScanner(f)
 	for s.Scan() {
-		result = append(result, s.Text())
+		result = append(result, strings.TrimSpace(s.Text()))
```

Comment:
Huff, are you sure about this use case ? 
Supporting files with trailing whitespace. I have rarely seen that being used intentionally. 

If the lines in a whitelist/blacklist file has trailing spaces those are  most probably left there unintentionally. 
I cannot think anyone benefiting from this "feature"